conf/application.conf

```diff
@@ -65,6 +65,12 @@ webKnossos {
     timeTrackingPause = 60 seconds
     inviteExpiry = 14 days
     ssoKey = ""
+    emailVerification {
+      activated = true
```

Comment:
should be false by default (for development)
(also reminder to reset mail.logToSstdout)

Reply:
(for the record, we agreed that activated=true makes sense, and shouldn’t block anyone while required is false)